daisy_workflows/build-publish/rhui/health_check.py

```diff
@@ -101,37 +140,21 @@ def main(node_type: str, result_file: typing.TextIO, nfs_mount: str):
     type=argparse.FileType("w"),
     help="file to write result",
   )
-  parser.add_argument(
-    "--nfs_mount",
-    dest="nfs_mount",
-    type=str,
-    nargs=1,
-    required=True,
-    help="directory of NFS mount",
-  )
-  parser.add_argument(
-    "--log",
-    dest="log",
-    type=str,
-    nargs=1,
-    choices=["stdout", "syslog"],
-    default="syslog",
-    help="where to write logs",
-  )
   args = parser.parse_args()
-  if args.log[0] == "stdout":
-    handler = logging.StreamHandler(sys.stdout)
-  else:
-    handler = logging.handlers.SysLogHandler(address="/dev/log")
-    # syslog parses on the colon to determine the tag for the message.
-    handler.ident = "rhui-health-check: "
+
+  handler = logging.StreamHandler(sys.stdout)
```

Comment:
If I recall, the syslog handler was helpful since it propagated the error level (so you can query it) and gave a more useful tag name.

Reply:
could be, but i don't think that is enough to merit using syslog directly. if you want to upgrade to structued cloud logging we can discuss that :)